test/Recipe-test.js

```diff
@@ -5,10 +5,58 @@ import ingredientsData from '../src/data/ingredients';
 
 
 describe('Recipe', () => {
-    let recipe;
+  let recipe;
 
-    beforeEach(() => {
-        recipe = new Recipe();
-    })
+  beforeEach(() => {
+    recipe = new Recipe(recipeData[0], ingredientsData);
+   
+  })
+  
+  it('should be a function', () => {
+    expect(Recipe).to.be.a('function');
+  });
+
+  it('should be an instance of Recipe Repository', () => {
+    expect(recipe).to.be.an.instanceof(Recipe);
+  });
+
+//   it.skip('should hold all the info for a recipe', () => {
+//     expect(recipe).to.deep.equal(recipeData[0]);
+//   })
+
+  it('should have an id', () => {
+    expect(recipe.id).to.deep.equal(recipeData[0].id)
+     
+  })
+
+  it('should have an ingredients property', () => {
+    expect(recipe.ingredients).to.deep.equal(recipeData[0].ingredients)
+  })
+
+  it('should have an instructions property', () => {
+    expect(recipe.instructions).to.deep.equal(recipeData[0].instructions)
+  })
+
+  it('should have a name property', () => {
+    expect(recipe.name).to.deep.equal(recipeData[0].name)
+  })
+
+  it('should have a tags property', () => {
+    expect(recipe.id).to.deep.equal(recipeData[0].id)
+  })
+
+  it('should have a method to determine the names of ingredients', () => {
```

Comment:
We will be building out a small, sample data set to use. When we have done that, we will know what values to be testing with.